CHANGELOG.md

```diff
@@ -56,15 +45,13 @@ This project does NOT adhere to [Semantic Versioning](https://semver.org/spec/v2
 
 ### Bug Fixes
 
-`[initializers]`
+`[create-sitecore-jss]` `[initializers]` 
```

Comment:
Only fixes that are present on current version (19.0.0) should be included in "Bug Fixes" list. All of these for create-sitecore-jss/initializers can be removed (it's a completely new feature in 20).